homeassistant/components/lamarzocco/config_flow.py

```diff
@@ -162,8 +166,10 @@ async def async_step_machine_selection(
 
             # validate local connection if host is provided
             if user_input.get(CONF_HOST):
+                if TYPE_CHECKING:
+                    assert self._client
```

Comment:
You can move the definition to a class attribute `_client: AsyncClient` under `VERSION = 2`
```suggestion
```

Reply:
true, I just don't like mixing class and instance attributes